.github/workflows/upgrade-test.yml

```diff
@@ -129,75 +85,51 @@ jobs:
           rename-to: try-runtime
           chmod: 0755
 
-      - name: Get upgrade-from-commit 📜
-        if: inputs.run-job
-        uses: ./.github/actions/get-workflow-commit
-        id: get-upgrade-from-commit
-        with:
-          workflow-name: release-${{ inputs.upgrade-from-release }}.yml
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: echo upgrade from commit 🔊
+      - name: Read old version from CFE upgrade-utils
         if: inputs.run-job
         run: |
-          echo "The upgrade-from-commit is: ${{ steps.get-upgrade-from-commit.outputs.commit-sha }}"
+          OLD_VERSION=$(grep -oP 'pub const OLD_VERSION: &str = "\K[^"]+' ./engine-upgrade-utils/src/lib.rs)
+          echo "OLD_VERSION=${OLD_VERSION}" >> $GITHUB_ENV
+          echo "OLD_VERSION=${OLD_VERSION}"
 
-      - name: Download latest release binaries 📥
+      - name: Download upgrade-from binaries from S3 📥
         if: inputs.run-job
-        uses: dawidd6/action-download-artifact@e7466d1a7587ed14867642c2ca74b5bcc1e19a2d
-        with:
-          workflow: release-${{ inputs.upgrade-from-release }}.yml
-          workflow_conclusion: ""
-          name: chainflip-backend-bin
-          github_token: ${{ secrets.CF_BACKEND_GITHUB_TOKEN }}
-          path: latest-release-bins
-          search_artifacts: true
+        run: |
+          mkdir -p upgrade-from-bins && cd upgrade-from-bins
+          curl https://artifacts.chainflip.io/${{ env.OLD_VERSION }}/chainflip-backend-bin.zip --output chainflip-backend-bin.zip
+          unzip chainflip-backend-bin.zip -d .
+          cd .. && ls -l upgrade-from-bins
 
       - name: Permissions for latest binaries 🛡️
         if: inputs.run-job
         run: |
-          chmod +x ./latest-release-bins/chainflip-*
-          chmod +x ./latest-release-bins/engine-runner
+          chmod +x ./upgrade-from-bins/chainflip-*
+          chmod +x ./upgrade-from-bins/engine-runner
 
-      - name: Get version of the latest release ✍️
+      - name: Get version of the upgrade-from binaries ✍️
         if: inputs.run-job
         run: |
           set -x
-          RELEASE_VERSION=$(./latest-release-bins/chainflip-node --version)
-          echo $RELEASE_VERSION
-          echo "RELEASE_VERSION=${RELEASE_VERSION}" >> $GITHUB_ENV
-
-      - name: Download upgrade-to binaries for push or pull request 📥
+          OLD_BIN_VERSION=$(./upgrade-from-bins/chainflip-node --version)
+          echo $OLD_BIN_VERSION
+          echo "OLD_BIN_VERSION=${OLD_BIN_VERSION}" >> $GITHUB_ENV
```

Comment:
can we store this value to `GITHUB_OUTPUT` too? would be handy if we'll need this info across other jobs in future